src/components/icons/PerronIcon.tsx

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import { ComponentWithAs } from '@chakra-ui/system';
+import { createIcon, IconProps } from '@chakra-ui/react';
+
+export const ChevronDownLargeBoldIcon: ComponentWithAs<'svg', IconProps> =
+  createIcon({
+    displayName: 'PerroIcon',
```

Comment:
Does it make sense to follow the same pattern here as in other icons? (E.g displayName: 'Perron' or displayName: 'ChevronDownLargeBold')

Reply:
Sorry, this icon doesn't exist, I just replace it with the one we already have